firmware/spade/src/rpi/audio.c

```diff
@@ -23,15 +23,15 @@ static struct audio_buffer_pool *audio_buffer_pool_init() {
     .sample_stride = 2
   };
 
-  struct audio_buffer_pool *producer_pool = audio_new_producer_pool(&producer_format, 3,
+  struct audio_buffer_pool  *producer_pool = audio_new_producer_pool(&producer_format, 2,
```

Comment:
This is a change to some audio code... the diff is all about sd card support - is this desired?

Reply:
Audio code changes are not desired and just there temporarily to free up memory needed for sd card. Freeing up memory elsewhere is something that I need to look into since this does make audio sound bad. From the numbers I last calculated, there shouldn't be too much more memory needed.

Reply:
Ahhhhhh... this is, therefore, an insurmountable (as of yet, w/ current code) technical issue - I understand, and it is good to know we have this problem.